GitUI/CommandsDialogs/SettingsDialog/CommonLogic.cs

```diff
@@ -20,12 +22,13 @@ public sealed class CommonLogic : Translate
         public readonly ConfigFileSettingsSet ConfigFileSettingsSet;
         public readonly GitModule Module;
 
+        public ConfigFileSettings EffectiveGlobalSettings => ConfigFileSettingsSet.EffectiveSettings.LowerPriority;
```

Comment:
It would be great to add a comment explaining this.

Reply:
I have created a function `ConfigFileSettings.GetEffectiveGlobalSettings()`.